crates/astria-celestia-client/src/client.rs

```diff
@@ -151,7 +151,6 @@ pub trait CelestiaClientExt: BlobClient {
     /// This calls the `blob.Submit` celestia-node RPC.
     async fn submit_sequencer_blocks(
         &self,
-        namespace: Namespace,
```

Comment:
I am not sure I like that the namespace is becoming an implicit function of the `blocks` sequence.

While I follow the thought process (coupling cometbft chain ID to namespace), it's now difficult to understand where the sequencer blocks will end up.

I prefer to make the namespace explicit and not couple so tightly to cometbft

Reply:
Discussed seperately offline. We want this tie, writing it somehwere different than specified by the block would be wrong. Adding logging + comments to address concerns over clarity.